ajax/libs/jquery-ui-bootstrap/package.json

```diff
@@ -7,12 +7,23 @@
     "jquery",
     "bootstrap"
   ],
-  "license": {
-    "name": "MIT License",
-    "url": "http://www.opensource.org/licenses/mit-license.php"
-  },
   "repository": {
     "type": "git",
     "url": "https://github.com/jquery-ui-bootstrap/jquery-ui-bootstrap.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jquery-ui-bootstrap/jquery-ui-bootstrap.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "css/custom-theme/jquery-ui*",
+          "third-party/**/*",
+          "assets/**/*"
```

Comment:
From some of the patterns, it looks like there may be something already hosted on CDNJS like font-awesome and bootstrap, jquery, html5shiv, etc. For those libraries, if they are not a specially modified versions, we shouldn't add them again. Please confirm that.